compiler/src/dotty/tools/dotc/parsing/Tokens.scala

```diff
@@ -273,7 +273,7 @@ object Tokens extends TokensCommon {
   final val closingRegionTokens = BitSet(RBRACE, RPAREN, RBRACKET, CASE) | statCtdTokens
 
   final val canStartIndentTokens: BitSet =
-    statCtdTokens | BitSet(COLONEOL, WITH, EQUALS, ARROW, CTXARROW, LARROW, WHILE, TRY, FOR, IF, THROW)
+    statCtdTokens | BitSet(COLONEOL, WITH, EQUALS, ARROW, CTXARROW, LARROW, WHILE, TRY, FOR, IF, THROW, RETURN)
```

Comment:
That's a global change, not only for the experimental flag? We should update it in Scalameta then, but should not be problematic.

Reply:
Yes, it's a global change

Reply:
https://github.com/scalameta/scalameta/pull/2264